app/models.py

```diff
@@ -334,7 +335,7 @@ class UpdateSessionAnswer(BaseModel):
     """Model for the body of the request that updates a session answer"""
 
     answer: Optional[answerType]
-    visited: Optional[Union[bool, None]]
+    visited: Optional[bool]
```

Comment:
is this the same as `visited: Optional[bool] = None`?

Reply:
yes.. but if `visited` is not in the `UpdateSessionAnswer` request body (sent by frontend), it gets removed -- via `remove_optional_unset_args()` in `routers/session_answers.py`